.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,27 @@
-<!--  Delete any sections below that are not relevant to this PR -->
+<!-- Insert GitHub issue number below -->
+Fixes #
 
 ## Description
-<!--- What are the changes? -->
+<!-- Briefly describe the proposed changes -->
 
-## Context
-<!--- Why did you make these changes, and why in this particular way? -->
+## Notes
+<!-- Be sure to mention anything unusual, out-of-scope or new technical debt, etc -->
 
-## References
-<!-- Include links to JIRA, Github, etc. if appropriate -->
+## Checklist
```

Comment:
Maybe move this to the bottom? After screenshots? This is mostly just for the person filing not for the people reviewing

Reply:
Yeah, I guess we could. Trouble is screenshots can take up a LOT of space, which means we have to scroll down all the way to bottom to review the checklist. The way I've done the checklist is that it's not just for the person filing, it also has a couple of at a glance things that are useful for reviewer too, such as the PR type (but this could be split out into separate PR templates - see other comment)

Reply:
Agreed, but from what I see other than the PR type, the rest is mostly a checklist that a b and c were run and done.

Reply:
Yeah, I acknowledge that I've veered away from the traditional idea of a PR checklist here. We just have to decide if we like this direction or not. I don't mind either way. Whatever works best for us.

Reply:
I think having a checklist to remind people that they ran a, b, and c is helpful and good to have. Sometimes doing many different PRs we forgot, so checking those makes us think again.
But I think most of the time it should be simply for the person writing the PR to check it off, not necessarily for the reviewers (other than to see it was checked and that those were completed) Ideally we just want to see:
- [X] I ran the build successfully and checked in all generated files. 
- [X] I verified test cases
- [X] Anythng else etc

And if those are all checked then we know they at least thought about them. But I think going into this thing of selecting what type of general fix this is or other things like that should probably go in the description or about this PR imo.